src/app/summary/page.tsx

```diff
@@ -0,0 +1,111 @@
+"use client";
+
+import { MiniPoster } from "@/components/title-page-components";
+import { dummyFilms } from "@/contexts/TitleContext";
+import { FactoidDto, PreviewTitleDto } from "@/dtos";
+import { dummyFactoids } from "@/dtos/factoid";
+import styled from "styled-components";
+
+const PageContainer = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	gap: 2rem;
+	margin-inline: 3rem;
+`;
+
+const HeaderBox = styled.div`
+	display: flex;
+	flex-direction: row;
+	gap: 1rem;
+	height: 10rem;
+`;
+
+const CardStyle = styled.div`
+	background-color: rgba(255, 255, 255, 0.1);
```

Comment:
Jasper always says we should use the `rgb` equivalent 😅 
```suggestion
	background-color: rgb(255 255 255 / 0.1);
```